src/app/containers/ConsentBanner/Banner/index.canonical.jsx

```diff
@@ -35,12 +35,21 @@ const Reject = (message, href, onClick, dataAttribute) => {
   );
 };
 
-const CanonicalConsentBannerContainer = ({ type, onReject, onAccept }) => {
+const CanonicalConsentBannerContainer = ({
+  type,
+  onReject,
+  onAccept,
+  consentBannerRef,
```

Comment:
can this ref be created internally within this container, I'm not sure it's needed outside of it?